ngraph/frontend/onnx_import/src/op/add.cpp

```diff
@@ -30,17 +31,19 @@ namespace ngraph
                 {
                     const Output<ngraph::Node> lhs_node = node.get_ng_inputs().at(0);
                     Output<ngraph::Node> rhs_node = node.get_ng_inputs().at(1);
-                    bool broadcast = node.get_attribute_value<std::int64_t>("broadcast", 0);
+                    const bool broadcast = node.get_attribute_value<std::int64_t>("broadcast", 0);
                     if (broadcast)
                     {
                         if (node.has_attribute("axis"))
                         {
                             // Unidirectional broadcast right node to left shape.
-                            auto axis = node.get_attribute_value<std::int64_t>("axis");
+                            const auto axis = node.get_attribute_value<std::int64_t>("axis");
+                            const auto axes_mapping = builder::opset1::get_axes_mapping_output(
```

Comment:
can axis be negative?
and int64_t to size_t conversion looks dangerous.

Reply:
No, ONNX opset 6 does not support negative axes.